en/performance/practical-search-performance-guide.md

```diff
@@ -235,12 +233,13 @@ that together define the behavior of a Vespa system:
 what functionality to use, the available document types, how ranking will be done
 and how data will be processed during feeding and indexing.
 Let's define the minimum set of required files to create our basic search application,
-which are `track.sd`, `services.xml`.
+which are `track.sd` and `services.xml`.
+Create directories for the configuration files:
 
 <div class="pre-parent">
   <button class="d-icon d-duplicate pre-copy-button" onclick="copyPreContent(this)"></button>
 <pre data-test="exec">
-$ mkdir app
+$ mkdir -p app/schemas; mkdir -p app/search/query-profiles/types
```

Comment:
Right, so the test framework handles this

Reply:
yes. but as the end user it was cumbersome to create all these dirs for the new files, and this simplified it